definitions/pipe.go

```diff
@@ -68,16 +67,6 @@ type NameReg interface {
 	Entries([]*event.FilterData) (*types.ResultListNames, error)
 }
 
-type Net interface {
```

Comment:
not used outside of v0, no need to have this on Pipe since underlying information replicated with Peers, Listeners etc. make way for RPC unification